backend/compact-connect/tests/unit/test_api.py

```diff
@@ -0,0 +1,107 @@
+import json
+from unittest import TestCase
+
+from aws_cdk.assertions import Template
+from aws_cdk.aws_apigateway import CfnResource, CfnMethod
+from aws_cdk.aws_lambda import CfnFunction
+
+from app import CompactConnectApp
+
+
+class TestApi(TestCase):
+    """
+    These tests are focused on checking that the API stack is configured correctly.
+
+    When adding or modifying API resources, a test should be added to ensure that the
+    resource is created as expected. The pattern for these tests includes the following checks:
+    1. The path and parent id of the API GW resource matches expected values.
```

Comment:
GW?

Reply:
It stands for Gateway. It's a common acronym used by AWS but I can spell out the word to avoid disambiguation.

Reply:
yeah let's spell it out just for clarity.